new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleOperatorValidation.java

```diff
@@ -65,69 +64,89 @@ public void testInstallingOperator() {
     // the kubernetes deployment.  this will complete quickly, and will either be
     // successful or not.
 
-    // get a new unique namespace
-    final String namespace = assertDoesNotThrow(TestActions::createUniqueNamespace,
+    // get a new unique opNamespace
+    final String opNamespace = assertDoesNotThrow(TestActions::createUniqueNamespace,
+        "Failed to create unique namespace due to ApiException");
+    logger.info(String.format("Got a new namespace called %s", opNamespace));
+
+    final String domainNamespace1 = assertDoesNotThrow(TestActions::createUniqueNamespace,
+        "Failed to create unique namespace due to ApiException");
+    logger.info(String.format("Got a new namespace called %s", domainNamespace1));
+
+    final String domainNamespace2 = assertDoesNotThrow(TestActions::createUniqueNamespace,
         "Failed to create unique namespace due to ApiException");
-    logger.info(String.format("Got a new namespace called %s", namespace));
+    logger.info(String.format("Got a new namespace called %s", domainNamespace2));
 
-    // Create a service account for the unique namespace
-    final String serviceAccountName = namespace + "-sa";
-    assertDoesNotThrow(
-        () -> TestActions.createServiceAccount(new V1ServiceAccount()
-            .metadata(new V1ObjectMeta().namespace(namespace).name(serviceAccountName))));
+    // Create a service account for the unique opNamespace
+    final String serviceAccountName = opNamespace + "-sa";
+    assertDoesNotThrow(() -> TestActions.createServiceAccount(new V1ServiceAccount()
+        .metadata(new V1ObjectMeta().namespace(opNamespace).name(serviceAccountName))));
```

Comment:
Every assertion should have a short message that would help someone debug the issue should it occur.

Reply:
will fix the assertions to have a message. Thanks @markxnelson for the assertions test in the other PR.

Reply:
done